lib/push/message_apns.rb

```diff
@@ -43,11 +45,32 @@ def alert
       MultiJson.load(string_or_json) rescue string_or_json
     end
 
-    # This method conforms to the enhanced binary format.
-    # http://developer.apple.com/library/ios/#documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/CommunicatingWIthAPS/CommunicatingWIthAPS.html#//apple_ref/doc/uid/TP40008194-CH101-SW4
+    def priority
+      if ( properties[:alert].nil? && 
+         properties[:badge].nil? && 
+         properties[:sound].nil? && 
+         properties[:expiry].nil? && 
```

Comment:
@jim0020 we don't need to check for a nil expiry........we just need to make sure that `alert`, `badge` and `sound` is nil. `expiry` is a required parameter on the message and will never be nil.

Can you remove this please?